yahoo_finance/__init__.py

```diff
@@ -176,13 +177,21 @@ def __init__(self, symbol):
         self._table = 'quotes'
         self._key = 'symbol'
         self.refresh()
+        if (self.validate() is False):
+            warnings.warn(symbol + " is not a valid symbol", RuntimeWarning, stacklevel=2)
 
     def _fetch(self):
         data = super(Share, self)._fetch()
         if data['LastTradeDate'] and data['LastTradeTime']:
             data[u'LastTradeDateTimeUTC'] = edt_to_utc('{0} {1}'.format(data['LastTradeDate'], data['LastTradeTime']))
         return data
 
+    def validate(self):
```

Comment:
Note that this function can be more succinctly written as the following:

```python
def validate(self):
    return (self.get_price() is not None)
```

Super simple implementation regardless, though. I like it!

cc @pdhimal1 @lukaszbanasiak